src/runtime/arc.ts

```diff
@@ -274,13 +274,18 @@ export class Arc {
           serializedData.forEach(a => {a.storageKey = storeId;});
         }
 
+        // eslint-disable-next-line require-atomic-updates
         context.resources += `resource ${id}Resource\n`;
         const indent = '  ';
+        // eslint-disable-next-line require-atomic-updates
         context.resources += indent + 'start\n';
 
         const data = JSON.stringify(serializedData);
+        // eslint-disable-next-line require-atomic-updates
         context.resources += data.split('\n').map(line => indent + line).join('\n');
+        // eslint-disable-next-line require-atomic-updates
         context.resources += '\n';
```

Comment:
How is this not atomic given JS single threaded nature? There is await here. Do you understand this?

Reply:
I tested this a bit and it appears that reading from context before an await and using it after triggers the warning.  So agreed that we should disable this; maybe the rule will get better in the future.